### DISCUSSION TITLE
Observability in Quarkus 3

### DISCUSSION THREAD

**Discussion by giscus** [quarkusio/quarkus#36161] [Quarkus Blog/Website]
# Observability in Quarkus 3

Quarkus: Supersonic Subatomic Java

https://quarkus.io/blog/quarkus-observability-3-3/

**Comment by Paun-Mihai** [comment]
Hello. Regarding the HttpServerMetricsTagsContributor, is there any similar functionality for the http clients? I would like to create tags based on a client request context.
